src/woo_search/search_index/api/serializers/publications.py

```diff
@@ -72,6 +73,42 @@ class DocumentSerializer(serializers.Serializer):
     )
 
 
+class DocumentIndexSerializer(DocumentSerializer):
+    download_url = serializers.URLField(
+        help_text=_(
+            "The URL to where the document can be downloaded from to index the contents."
+        ),
+        write_only=True,
+        required=False,
+        allow_null=True,
+        default=None,
+    )
+    file_size = serializers.IntegerField(
+        help_text=_("The size of the document file on disk, in bytes."),
+        write_only=True,
+        required=False,
+        allow_null=True,
+        default=None,
+    )
```

Comment:
Should combine this field in a separate serializer so it is easier to understand that these fields are for the document upload?